python/cudf/cudf/core/series.py

```diff
@@ -3812,22 +3890,19 @@ def isocalendar(self):
         0    2009
         1    <NA>
         Name: year, dtype: object
-        """
+        """  # noqa: E501
         return cudf.core.tools.datetimes._to_iso_calendar(self)
 
     @property  # type: ignore
     @_cudf_nvtx_annotate
     def is_month_start(self):
-        """
-        Booleans indicating if dates are the first day of the month.
-        """
+        """Booleans indicating if dates are the first day of the month."""
```

Comment:
```suggestion
        """Indicates whether the date is the first day of the month."""
```
https://pandas.pydata.org/docs/reference/api/pandas.Series.dt.is_month_start.html

Reply:
This almost works, I'll probably just have to use "Indicate" instead of "Indicates".